source/ghost-text.js

```diff
@@ -68,6 +68,12 @@ class AdvancedTextWrapper {
 }
 
 function wrapField(field) {
+	const monaco = field.closest('.monaco-editor');
+	if (monaco) {
+		const visualElement = monaco.querySelector('.monaco-editor-background');
+		return new AdvancedTextWrapper(monaco, visualElement);
```

Comment:
This works now

<img width="635" alt="Screenshot 3" src="https://user-images.githubusercontent.com/1402241/215079115-efa6ac9f-0296-4238-8d16-d07398c959be.png">